src/test/java/games/strategy/engine/framework/ArgParserTest.java

```diff
@@ -75,6 +76,15 @@ public void singleUrlArgIsAssumedToBeMapDownloadProperty() {
   }
 
   @Test
+  public void singleUrlArgIsUrlDecoded() throws UnsupportedEncodingException {
+    final String testUrl = "triplea:Something%20with+spaces%20and%20Special%20chars%20%F0%9F%A4%94";
```

Comment:
I added an + on purpose, because we probably also want to support URIs with + chars instead of spaces